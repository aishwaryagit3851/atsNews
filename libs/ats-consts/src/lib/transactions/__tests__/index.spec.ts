import {
  TRANSACTIONS_TYPES_OBJ,
  TRANSACTIONS_TYPES,
  TRANSACTIONS_STATUS_OBJ,
  TRANSACTIONS_STATUS,
  TRANSACTIONS_METHODS_OBJ,
  TRANSACTIONS_METHODS,
  VOUCHER_TYPES_OBJ,
  VOUCHER_TYPES,
  PURCHASE_TYPES,
  PURCHASE_TYPE_OBJ,
  INVOICE_TYPE_OBJ,
  INVOICE_TYPES
} from '..';

describe('ATS CONSTS for - transactions', () => {
  it('Match consts to snapshot', () => {
    expect(TRANSACTIONS_TYPES_OBJ).toMatchSnapshot();
    expect(TRANSACTIONS_TYPES).toMatchSnapshot();

    expect(TRANSACTIONS_STATUS_OBJ).toMatchSnapshot();
    expect(TRANSACTIONS_STATUS).toMatchSnapshot();

    expect(TRANSACTIONS_METHODS_OBJ).toMatchSnapshot();
    expect(TRANSACTIONS_METHODS).toMatchSnapshot();

    expect(VOUCHER_TYPES_OBJ).toMatchSnapshot();
    expect(VOUCHER_TYPES).toMatchSnapshot();

    expect(PURCHASE_TYPE_OBJ).toMatchSnapshot();
    expect(PURCHASE_TYPES).toMatchSnapshot();

    expect(INVOICE_TYPE_OBJ).toMatchSnapshot();
    expect(INVOICE_TYPES).toMatchSnapshot();
  });
});
