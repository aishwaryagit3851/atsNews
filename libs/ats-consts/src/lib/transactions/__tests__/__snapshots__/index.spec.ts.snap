// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ATS CONSTS for - transactions Match consts to snapshot 1`] = `
{
  "CR": "CR",
  "DR": "DR",
}
`;

exports[`ATS CONSTS for - transactions Match consts to snapshot 2`] = `
[
  "CR",
  "DR",
]
`;

exports[`ATS CONSTS for - transactions Match consts to snapshot 3`] = `
{
  "DONE": "DONE",
  "DUE": "DUE",
  "PARTIAL": "PARTIAL",
}
`;

exports[`ATS CONSTS for - transactions Match consts to snapshot 4`] = `
[
  "DUE",
  "PARTIAL",
  "DONE",
]
`;

exports[`ATS CONSTS for - transactions Match consts to snapshot 5`] = `
{
  "BANK_TRANSFER": "BANK_TRANSFER",
  "CARD": "CARD",
  "CASH": "CASH",
  "CHEQUE": "CHEQUE",
}
`;

exports[`ATS CONSTS for - transactions Match consts to snapshot 6`] = `
[
  "CASH",
  "CARD",
  "CHEQUE",
  "BANK_TRANSFER",
]
`;

exports[`ATS CONSTS for - transactions Match consts to snapshot 7`] = `
{
  "CREDIT_NOTE": "CREDIT_NOTE",
  "DEBIT_NOTE": "DEBIT_NOTE",
  "JOURNAL": "JOURNAL",
  "OPENING_BALANCE": "OPENING_BALANCE",
  "PAYMENT": "PAYMENT",
  "PURCHASE": "PURCHASE",
  "PURCHASE_RETURN": "PURCHASE_RETURN",
  "RECIPT": "RECIPT",
  "SALES": "SALES",
  "SALES_RETURN": "SALES_RETURN",
}
`;

exports[`ATS CONSTS for - transactions Match consts to snapshot 8`] = `
[
  "PAYMENT",
  "RECIPT",
  "SALES",
  "PURCHASE",
  "SALES_RETURN",
  "PURCHASE_RETURN",
  "CREDIT_NOTE",
  "DEBIT_NOTE",
  "JOURNAL",
  "OPENING_BALANCE",
]
`;

exports[`ATS CONSTS for - transactions Match consts to snapshot 9`] = `
{
  "CAPITAL": "CAPITAL",
  "INPUT": "INPUT",
}
`;

exports[`ATS CONSTS for - transactions Match consts to snapshot 10`] = `
[
  "INPUT",
  "CAPITAL",
]
`;

exports[`ATS CONSTS for - transactions Match consts to snapshot 11`] = `
{
  "B2B": "B2B",
  "B2C": "B2C",
}
`;

exports[`ATS CONSTS for - transactions Match consts to snapshot 12`] = `
[
  "B2B",
  "B2C",
]
`;
